description.md

```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easily digestible format for presenting information.
+As such, media is typically embbeded within the text that is destined reading, slides, demos, practice items and projects.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
+#### Images
+
+For the creation of images and diagrams we use [draw.io](draw.io).
+`draw.io` is a free, easy to use tool which is available in the browser.
```

Comment:
Argue why having it as a browser helps, that it exports SVG and that it contains many useful icons / basic blocks. Hence, our decision to use it.